Sun Aug 11 18:37:53 BRT 2019 Thread[Thread-6,6,main] Cleanup action starting
java.sql.SQLException: Database 'C:\000000\JM\00_Generico\010_Generico_JM_Projeto_Produtos\Generico_JM_010_Produtos/Banco_De_Dados' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at login_do_sistema.Login_Do_Sistema.login(Login_Do_Sistema.java:793)
	at login_do_sistema.Login_Do_Sistema.verificar_Usuario(Login_Do_Sistema.java:532)
	at login_do_sistema.Login_Do_Sistema.access$3200(Login_Do_Sistema.java:38)
	at login_do_sistema.Login_Do_Sistema$16.run(Login_Do_Sistema.java:514)
Caused by: java.sql.SQLException: Database 'C:\000000\JM\00_Generico\010_Generico_JM_Projeto_Produtos\Generico_JM_010_Produtos/Banco_De_Dados' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 18 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Database 'C:\000000\JM\00_Generico\010_Generico_JM_Projeto_Produtos\Generico_JM_010_Produtos/Banco_De_Dados' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at login_do_sistema.Login_Do_Sistema.login(Login_Do_Sistema.java:793)
	at login_do_sistema.Login_Do_Sistema.verificar_Usuario(Login_Do_Sistema.java:532)
	at login_do_sistema.Login_Do_Sistema.access$3200(Login_Do_Sistema.java:38)
	at login_do_sistema.Login_Do_Sistema$16.run(Login_Do_Sistema.java:514)
============= end nested exception, level (1) ===========
Cleanup action completed
----------------------------------------------------------------
Sun Aug 11 18:38:14 BRT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.8.2.2 - (1181258): instance a816c00e-016c-829d-c4b9-00000076d3e0 
on database directory D:\_01\00_Generico\009_Generico_JM_Projeto_Produtos\Generico_JM_009_Produtos\Banco_De_Dados  with class loader sun.misc.Launcher$AppClassLoader@14991ad 
Loaded from file:/D:/_01/00_Generico/009_Generico_JM_Projeto_Produtos/Generico_JM_009_Produtos/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=D:\_01\00_Generico\009_Generico_JM_Projeto_Produtos\Generico_JM_009_Produtos
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
